src/librustc_back/target/dragonfly_base.rs

```diff
@@ -16,20 +16,10 @@ pub fn opts() -> TargetOptions {
         linker: "cc".to_string(),
         dynamic_linking: true,
         executables: true,
-        linker_is_gnu: true,
```

Comment:
Just confirming, but it looks like the only place this is used is passing the `-O1` flag to the linker, and the dragonfly linker doesn't support that?